front/src/modules/timetableItem/components/ImportTimetableItem/helpers/parseXML.ts

```diff
@@ -138,11 +347,11 @@ const parseXML = async (xmlDoc: Document): Promise<TimetableJsonPayload> => {
       train_name: trainNumber,
       rolling_stock_name: rollingStockName || formationRef || '', // RollingStocks in xml files rarely have the correct format
       start_time: new Date(`${startDate} ${firstDepartureTimeformatted}`).toISOString(),
-      constraint_distribution: xmlDoc.documentElement.hasAttribute('xmlns:tps')
-        ? 'STANDARD'
-        : 'MARECO',
+      constraint_distribution: 'STANDARD',
       path,
       schedule,
+      margins: { boundaries: [], values: ['0%'] },
+      power_restrictions: [],
```

Comment:
Why do we need to add these fields?

Reply:
Hm, I'm confused. The import code was working fine before this PR without these two fields. What has changed to make it so the API rejects payloads without these fields now?